### ISSUE TITLE
Future<dynamic> is not subtype of Future<String> at requestWillBeSent

### ISSUE THREAD

**Issue by itsJoKr** [brianegan/flutter_stetho#1]
I've just implemented Stetho in this one project. Requests are failing due to issue:

```
type 'Future<dynamic>' is not a subtype of type 'Future<String>' where
  Future is from dart:async
  Future is from dart:async
```
```
#0      MethodChannelController.responseHeadersReceived (package:flutter_stetho/src/method_channel_controller.dart:17:16)
#1      StethoHttpClientRequest.close (package:flutter_stetho/src/http_client_request.dart:62:29)
<asynchronous suspension>
#2      Stream.pipe.<anonymous closure> (dart:async/stream.dart:609:70)
#3      _RootZone.runUnary (dart:async/zone.dart:1381:54)
#4      _FutureListener.handleValue (dart:async/future_impl.dart:129:18)
#5      Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:633:45)
#6      Future._propagateToListeners (dart:async/future_impl.dart:662:32)
#7      Future._completeWithValue (dart:async/future_impl.dart:477:5)
#8      Future._asyncComplete.<anonymous closure> (dart:async/future_impl.dart:507:7)
#9      _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
#10     _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
```

Can't really help much. The `requestWillBeSent` method expect Future\<String\>, but it does't get that. First time using this package. Any idea why is this happening?

**Comment by brianegan** [comment]
My gosh, somehow wasn't watching the repo and missed your message. My bad...

Luckily, @thejakeofink has a fix up. I'll verify that and publish a new version.

**Comment by brianegan** [comment]
Fix published as `0.1.1`.
